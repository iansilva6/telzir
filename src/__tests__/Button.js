import React from 'react';
import DefaultButton from '../components/Button/';
import renderer from 'react-test-renderer';

describe("Button Component", ()=>{
    it("renders as expected", ()=>{
        const tree = renderer
            .create(<DefaultButton
                        text={"Texto"}
                    />)
            .toJSON();

        expect(tree).toMatchSnapshot();
    })
});