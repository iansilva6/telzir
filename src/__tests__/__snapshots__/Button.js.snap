// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component renders as expected 1`] = `
<button
  className="sc-AxjAm duCwob"
  onClick={[Function]}
>
  Texto
</button>
`;
